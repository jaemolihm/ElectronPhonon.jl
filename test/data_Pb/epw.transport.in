--
&inputepw
  prefix      = 'pb'
  amass(1)    = 207.2
  outdir      = './temp/'
  dvscf_dir   = './save/'
  etf_mem     = 0

  elph        = .true.
  kmaps       = .true.  ! if true, read kmaps from file
  epbread     = .false.
  epwread     = .true.

  lpolar      = .false.
  lifc        = .true.
  use_ws      = .true.
  vme         = .true.

  nbndsub     =  4
  bands_skipped = 'exclude_bands = 1-5'

  wannierize  = .false.

  fsthick      = 0.5 ! eV
  degaussw     = 0.05 ! eV
  assume_metal = .true.
  ngaussw      = -99  ! we want F-D distribution for metals

  int_mob       = .true.
  iterative_bte = .true.
  scattering    = .true.
  carrier       = .false.
  mp_mesh_k     = .true.
  epmatkqread   = .false.
  mob_maxiter   = 30
  broyden_beta  = 1.0

  nstemp        = 2
  temps         = 100 200 300

  nkf1        = 10
  nkf2        = 10
  nkf3        = 10
  nqf1        = 10
  nqf2        = 10
  nqf3        = 10

  nk1         = 3
  nk2         = 3
  nk3         = 3

  nq1         = 3
  nq2         = 3
  nq3         = 3
 /
