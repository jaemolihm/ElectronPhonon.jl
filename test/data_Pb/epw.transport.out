                                                                                      
                                       ``:oss/                                        
                           `.+s+.     .+ys--yh+     `./ss+.                           
                          -sh//yy+`   +yy   +yy    -+h+-oyy                           
                          -yh- .oyy/.-sh.   .syo-.:sy-  /yh                           
                 `.-.`    `yh+   -oyyyo.     `/syys:    oys      `.`                  
               `/+ssys+-` `sh+      `                   oys`   .:osyo`                
               -yh- ./syyooyo`                          .sys+/oyo--yh/                
               `yy+    .-:-.                             `-/+/:`  -sh-                
                /yh.                                              oys                 
          ``..---hho---------`   .---------..`      `.-----.`    -hd+---.             
       `./osmNMMMMMMMMMMMMMMMs. +NNMMMMMMMMNNmh+.   yNMMMMMNm-  oNMMMMMNmo++:`        
       +sy--/sdMMMhyyyyyyyNMMh- .oyNMMmyyyyyhNMMm+` -yMMMdyyo:` .oyyNMMNhs+syy`       
       -yy/   /MMM+.`-+/``mMMy-   `mMMh:`````.dMMN:` `MMMy-`-dhhy```mMMy:``+hs        
        -yy+` /MMMo:-mMM+`-oo/.    mMMh:     `dMMN/`  dMMm:`dMMMMy..MMMo-.+yo`        
         .sys`/MMMMNNMMMs-         mMMmyooooymMMNo:   oMMM/sMMMMMM++MMN//oh:          
          `sh+/MMMhyyMMMs- `-`     mMMMMMMMMMNmy+-`   -MMMhMMMsmMMmdMMd/yy+           
    `-/+++oyy-/MMM+.`/hh/.`mNm:`   mMMd+/////:-.`      NMMMMMd/:NMMMMMy:/yyo/:.`      
   +os+//:-..-oMMMo:--:::-/MMMo. .-mMMd+---`           hMMMMN+. oMMMMMo. `-+osyso:`   
   syo     `mNMMMMMNNNNNNNNMMMo.oNNMMMMMNNNN:`         +MMMMs:`  dMMMN/`     ``:syo   
   /yh`     :syyyyyyyyyyyyyyyy+.`+syyyyyyyyo:`         .oyys:`   .oyys:`        +yh   
   -yh-        ````````````````    `````````              ``        ``          oys   
   -+h/------------------------::::::::://////++++++++++++++++++++++///////::::/yd:   
   shdddddddddddddddddddddddddddddhhhhhhhhyyyyyssssssssssssssssyyyyyyyhhhhhhhddddh`   
                                                                                      
  S. Ponce, E. R. Margine, C. Verdi, and F. Giustino,                                 
                                                Comput. Phys. Commun. 209, 116 (2016) 
                                                                                      

     Program EPW v.5.3.1 starts on 10Sep2021 at 13:56: 5 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     1 processors

     MPI processes distributed on     1 nodes
     Reading input from epw.transport.in

     WARNING: Mismatch between temps(:) and nstemp

     WARNING: Using supplied temperature list and ignoring nstemp
                      
     ------------------------------------------------------------------------ 
                   RESTART - RESTART - RESTART - RESTART                         
     Restart is done without reading PWSCF save file.                  
     Be aware that some consistency checks are therefore not done.                  
     ------------------------------------------------------------------------ 
                      

     --                                                                         

     bravais-lattice index     =            0
     lattice parameter (a_0)   =       0.0000  a.u.
     unit-cell volume          =       0.0000 (a.u.)^3
     number of atoms/cell      =            0
     number of atomic types    =            0
     kinetic-energy cut-off    =       0.0000  Ry
     charge density cut-off    =       0.0000  Ry
     Exchange-correlation= not set
                           (  -1  -1  -1  -1  -1  -1  -1)


     celldm(1)=    0.00000  celldm(2)=    0.00000  celldm(3)=    0.00000
     celldm(4)=    0.00000  celldm(5)=    0.00000  celldm(6)=    0.00000

     crystal axes: (cart. coord. in units of a_0)
               a(1) = (  0.0000  0.0000  0.0000 )  
               a(2) = (  0.0000  0.0000  0.0000 )  
               a(3) = (  0.0000  0.0000  0.0000 )  

     reciprocal axes: (cart. coord. in units 2 pi/a_0)
               b(1) = (  0.0000  0.0000  0.0000 )  
               b(2) = (  0.0000  0.0000  0.0000 )  
               b(3) = (  0.0000  0.0000  0.0000 )  


     Atoms inside the unit cell: 

   Cartesian axes

     site n.  atom      mass           positions (a_0 units)

 
     No symmetry!

     G cutoff =    0.0000  (      0 G-vectors)     FFT grid: (  0,  0,  0)
     number of k points=    0
                       cart. coord. in units 2pi/a_0
     EPW          :      0.00s CPU      0.00s WALL

     EPW          :      0.00s CPU      0.00s WALL


     -------------------------------------------------------------------
     Using pb.ukk from disk
     -------------------------------------------------------------------

     Symmetries of Bravais lattice:  48
     Symmetries of crystal:          48

     Do not need to read .epb files; read .fmt files


     Band disentanglement is used: nbndsub =    4
     Construct the Wigner-Seitz cell using Wannier centers and atomic positions 
     Number of WS vectors for electrons       43
     Number of WS vectors for phonons       43
     Number of WS vectors for electron-phonon       43
     Maximum number of cores for efficient parallelization       43

     Reading Hamiltonian, Dynamical matrix and EP vertex in Wann rep from file


     Reading interatomic force constants

     IFC last    0.0006587
      Imposed simple ASR

     Finished reading ifcs


     Finished reading Wann rep data from file

     ===================================================================
     Memory usage:  VmHWM =        42Mb
                   VmPeak =      2591Mb
     ===================================================================
     
     Using uniform q-mesh:   10  10  10
     Size of q point mesh for interpolation:       1000
     Using uniform MP k-mesh:   10  10  10
     Size of k point mesh for interpolation:         94
     Max number of k points per pool:               94

     Fermi energy coarse grid =   9.843414 eV

     Skipping the first    5 bands:

     The Fermi level will be determined with   4.00000 electrons

     Fermi energy is calculated from the fine k-mesh: Ef =  11.594123 eV

     Warning: check if difference with Fermi level fine grid makes sense

     ===================================================================

              ibndmin =     2  ebndmin =    11.115 eV
              ibndmax =     3  ebndmax =    11.888 eV


     Number of ep-matrix elements per pool :          564 ~=    4.41 Kb (@ 8 bytes/ DP)

     File ./Fepmatkq1/pb.epmatkq1_0 deleted, as requested

     File ./Fsparse/sparse_0 deleted, as requested
      
     A selecq.fmt file was found but re-created because selecqread == .FALSE. 
     Number selected, total            100            138
     Number selected, total            200            271
     Number selected, total            300            394
     Number selected, total            400            529
     Number selected, total            500            658
     Number selected, total            600            785
     Number selected, total            700            936
     We only need to compute      723 q-points
      

     ===================================================================
     Scattering rate for IBTE
     ===================================================================

     Restart and restart_step inputs deactivated (restart point at every q-points).
     No intermediate mobility will be shown.

     Fermi Surface thickness =   0.500000 eV
     This is computed with respect to the fine Fermi level  11.594123 eV
     Only states between  11.094123 eV and  12.094123 eV will be included

     Save matrix elements larger than threshold:   0.250000000000E-22
      
     Progression iq (fine) =        100/       723
     Progression iq (fine) =        200/       723
     Progression iq (fine) =        300/       723
     Progression iq (fine) =        400/       723
     Progression iq (fine) =        500/       723
     Progression iq (fine) =        600/       723
     Progression iq (fine) =        700/       723
      
     epmatkqread automatically changed to .TRUE. as all scattering have been computed.
      
     ===================================================================
     Memory usage:  VmHWM =        46Mb
                   VmPeak =      2707Mb
     ===================================================================

     Number of elements per core       1759
     Symmetry mapping finished
      
     =============================================================================================
     BTE in the self-energy relaxation time approximation (SERTA)
     =============================================================================================

     =============================================================================================
       Temp     Fermi        DOS        Population SR                 Conductivity 
        [K]      [eV]    [states/Ry] [carriers per cell]               [Ohm.cm]^-1 
     =============================================================================================

      100.000  11.4498   0.37826E+01   0.00000E+00    0.966388E+05    0.160271E-14    0.804560E-12
                                                      0.000000E+00    0.966388E+05    0.000000E+00
                                                      0.810970E-12    0.160271E-14    0.966388E+05
      200.000  11.4799   0.26985E+01  -0.91593E-15    0.214829E+05    0.000000E+00   -0.160302E-13
                                                     -0.320542E-14    0.214829E+05    0.961625E-14
                                                      0.961312E-14    0.000000E+00    0.214829E+05
      300.000  11.5133   0.16747E+01  -0.91593E-15    0.843989E+04    0.320542E-14   -0.641083E-14
                                                     -0.320542E-14    0.843989E+04    0.320542E-14
                                                     -0.192325E-13   -0.320542E-14    0.843989E+04
      
     =============================================================================================
     Start solving iterative Boltzmann Transport Equation
     =============================================================================================

     Iteration number:         1

     =============================================================================================
       Temp     Fermi        DOS        Population SR                 Conductivity 
        [K]      [eV]    [states/Ry] [carriers per cell]               [Ohm.cm]^-1 
     =============================================================================================

      100.000  11.4498   0.37826E+01   0.88821E-15    0.107119E+06    0.512867E-13    0.974445E-12
                                                     -0.512867E-13    0.107119E+06    0.512867E-13
                                                      0.820587E-12    0.512867E-13    0.107119E+06
      200.000  11.4799   0.26985E+01  -0.77714E-15    0.239672E+05    0.000000E+00    0.320542E-12
                                                      0.000000E+00    0.239672E+05    0.000000E+00
                                                      0.205147E-12    0.000000E+00    0.239672E+05
      300.000  11.5133   0.16747E+01  -0.22237E-15    0.943501E+04   -0.128217E-13    0.769300E-13
                                                     -0.641083E-14    0.943501E+04   -0.641083E-14
                                                     -0.640833E-13    0.000000E+00    0.943501E+04

                                                      0.107119E+06    Max error
     Iteration number:         2

     =============================================================================================
       Temp     Fermi        DOS        Population SR                 Conductivity 
        [K]      [eV]    [states/Ry] [carriers per cell]               [Ohm.cm]^-1 
     =============================================================================================

      100.000  11.4498   0.37826E+01   0.14877E-13    0.109808E+06    0.000000E+00    0.615440E-12
                                                      0.102573E-12    0.109808E+06    0.000000E+00
                                                      0.666727E-12    0.000000E+00    0.109808E+06
      200.000  11.4799   0.26985E+01  -0.61063E-14    0.245581E+05    0.256433E-13   -0.128220E-12
                                                     -0.384650E-13    0.245581E+05    0.128217E-13
                                                     -0.230793E-12    0.000000E+00    0.245581E+05
      300.000  11.5133   0.16747E+01   0.55511E-16    0.966437E+04   -0.128217E-13   -0.512867E-13
                                                      0.000000E+00    0.966437E+04    0.000000E+00
                                                     -0.108984E-12    0.128217E-13    0.966437E+04

                                                      0.268882E+04    Max error
     Iteration number:         3

     =============================================================================================
       Temp     Fermi        DOS        Population SR                 Conductivity 
        [K]      [eV]    [states/Ry] [carriers per cell]               [Ohm.cm]^-1 
     =============================================================================================

      100.000  11.4498   0.37826E+01   0.22204E-15    0.109939E+06   -0.512867E-13    0.256433E-12
                                                      0.153860E-12    0.109939E+06    0.512867E-13
                                                      0.102573E-12    0.512867E-13    0.109939E+06
      200.000  11.4799   0.26985E+01   0.88818E-15    0.245959E+05   -0.128217E-13   -0.256430E-12
                                                     -0.512867E-13    0.245959E+05    0.000000E+00
                                                     -0.282074E-12    0.128217E-13    0.245959E+05
      300.000  11.5133   0.16747E+01  -0.33339E-15    0.968010E+04    0.128217E-13   -0.962126E-13
                                                      0.192325E-13    0.968010E+04   -0.192325E-13
                                                     -0.834160E-13   -0.128217E-13    0.968010E+04

                                                      0.131322E+03    Max error
     Iteration number:         4

     =============================================================================================
       Temp     Fermi        DOS        Population SR                 Conductivity 
        [K]      [eV]    [states/Ry] [carriers per cell]               [Ohm.cm]^-1 
     =============================================================================================

      100.000  11.4498   0.37826E+01  -0.71054E-14    0.110040E+06    0.256433E-13    0.461581E-12
                                                      0.000000E+00    0.110040E+06    0.000000E+00
                                                      0.718013E-12   -0.256433E-13    0.110040E+06
      200.000  11.4799   0.26985E+01  -0.11104E-15    0.246177E+05   -0.256433E-13   -0.320542E-12
                                                     -0.128217E-13    0.246177E+05   -0.128217E-13
                                                     -0.371828E-12    0.256433E-13    0.246177E+05
      300.000  11.5133   0.16747E+01   0.11156E-15    0.968853E+04   -0.641083E-14    0.577977E-13
                                                      0.641083E-14    0.968853E+04   -0.192325E-13
                                                      0.448508E-13    0.641083E-14    0.968853E+04

                                                      0.101297E+03    Max error
     Iteration number:         5

     =============================================================================================
       Temp     Fermi        DOS        Population SR                 Conductivity 
        [K]      [eV]    [states/Ry] [carriers per cell]               [Ohm.cm]^-1 
     =============================================================================================

      100.000  11.4498   0.37826E+01  -0.84703E-21    0.110035E+06   -0.512867E-13    0.307720E-12
                                                     -0.153860E-12    0.110035E+06    0.512867E-13
                                                     -0.153860E-12   -0.102573E-12    0.110035E+06
      200.000  11.4799   0.26985E+01   0.00000E+00    0.246169E+05    0.384650E-13   -0.769300E-13
                                                      0.256433E-13    0.246169E+05    0.000000E+00
                                                     -0.769300E-13   -0.128217E-13    0.246169E+05
      300.000  11.5133   0.16747E+01  -0.44409E-15    0.968825E+04   -0.128217E-13   -0.385151E-13
                                                      0.000000E+00    0.968825E+04    0.000000E+00
                                                     -0.128267E-12    0.000000E+00    0.968825E+04

                                                      0.558554E+01    Max error
     Iteration number:         6

     =============================================================================================
       Temp     Fermi        DOS        Population SR                 Conductivity 
        [K]      [eV]    [states/Ry] [carriers per cell]               [Ohm.cm]^-1 
     =============================================================================================

      100.000  11.4498   0.37826E+01  -0.24425E-14    0.110040E+06    0.000000E+00    0.138474E-11
                                                     -0.153860E-12    0.110040E+06   -0.512867E-13
                                                      0.107702E-11   -0.512867E-13    0.110040E+06
      200.000  11.4799   0.26985E+01   0.00000E+00    0.246180E+05    0.000000E+00    0.153857E-12
                                                      0.000000E+00    0.246180E+05    0.000000E+00
                                                      0.230787E-12    0.000000E+00    0.246180E+05
      300.000  11.5133   0.16747E+01  -0.11102E-15    0.968867E+04    0.128217E-13    0.320542E-13
                                                     -0.641083E-14    0.968867E+04   -0.192325E-13
                                                      0.320542E-13    0.000000E+00    0.968867E+04

                                                      0.504973E+01    Max error
     Iteration number:         7

     =============================================================================================
       Temp     Fermi        DOS        Population SR                 Conductivity 
        [K]      [eV]    [states/Ry] [carriers per cell]               [Ohm.cm]^-1 
     =============================================================================================

      100.000  11.4498   0.37826E+01   0.19984E-14    0.110039E+06    0.000000E+00    0.359007E-12
                                                      0.000000E+00    0.110039E+06    0.000000E+00
                                                      0.102573E-11    0.000000E+00    0.110039E+06
      200.000  11.4799   0.26985E+01  -0.45519E-14    0.246178E+05    0.128217E-13    0.256430E-12
                                                      0.384650E-13    0.246178E+05   -0.128217E-13
                                                      0.230787E-12    0.128217E-13    0.246178E+05
      300.000  11.5133   0.16747E+01  -0.55620E-16    0.968861E+04    0.641083E-14   -0.641083E-14
                                                      0.000000E+00    0.968861E+04    0.000000E+00
                                                      0.384400E-13   -0.641083E-14    0.968861E+04

                                                      0.802069E+00    Max error
     Iteration number:         8

     =============================================================================================
       Temp     Fermi        DOS        Population SR                 Conductivity 
        [K]      [eV]    [states/Ry] [carriers per cell]               [Ohm.cm]^-1 
     =============================================================================================

      100.000  11.4498   0.37826E+01  -0.13545E-13    0.110039E+06   -0.512867E-13    0.307720E-12
                                                     -0.512867E-13    0.110039E+06   -0.512867E-13
                                                      0.871874E-12   -0.102573E-12    0.110039E+06
      200.000  11.4799   0.26985E+01  -0.44408E-15    0.246179E+05    0.000000E+00    0.320542E-12
                                                      0.000000E+00    0.246179E+05    0.000000E+00
                                                      0.282077E-12    0.000000E+00    0.246179E+05
      300.000  11.5133   0.16747E+01  -0.22194E-15    0.968863E+04    0.641083E-14    0.449259E-13
                                                      0.000000E+00    0.968863E+04    0.000000E+00
                                                      0.770051E-13    0.641083E-14    0.968863E+04

                                                      0.309844E+00    Max error
     Iteration number:         9

     =============================================================================================
       Temp     Fermi        DOS        Population SR                 Conductivity 
        [K]      [eV]    [states/Ry] [carriers per cell]               [Ohm.cm]^-1 
     =============================================================================================

      100.000  11.4498   0.37826E+01  -0.19984E-14    0.110039E+06    0.000000E+00    0.666727E-12
                                                      0.000000E+00    0.110039E+06    0.000000E+00
                                                      0.359006E-12    0.000000E+00    0.110039E+06
      200.000  11.4799   0.26985E+01   0.67763E-19    0.246179E+05    0.000000E+00   -0.307720E-12
                                                      0.000000E+00    0.246179E+05    0.000000E+00
                                                     -0.141038E-12    0.000000E+00    0.246179E+05
      300.000  11.5133   0.16747E+01  -0.38869E-15    0.968863E+04   -0.641083E-14   -0.121806E-12
                                                      0.128217E-13    0.968863E+04    0.128217E-13
                                                     -0.961375E-13   -0.641083E-14    0.968863E+04

                                                      0.692919E-01    Max error
     Iteration number:        10

     =============================================================================================
       Temp     Fermi        DOS        Population SR                 Conductivity 
        [K]      [eV]    [states/Ry] [carriers per cell]               [Ohm.cm]^-1 
     =============================================================================================

      100.000  11.4498   0.37826E+01  -0.14211E-13    0.110039E+06   -0.769300E-13   -0.564153E-12
                                                      0.102573E-12    0.110039E+06    0.000000E+00
                                                     -0.153860E-12    0.179503E-12    0.110039E+06
      200.000  11.4799   0.26985E+01  -0.66615E-15    0.246179E+05    0.128217E-13    0.333366E-12
                                                      0.384650E-13    0.246179E+05   -0.128217E-13
                                                      0.230793E-12    0.128217E-13    0.246179E+05
      300.000  11.5133   0.16747E+01   0.16653E-15    0.968863E+04    0.000000E+00   -0.500846E-16
                                                      0.000000E+00    0.968863E+04    0.000000E+00
                                                     -0.705943E-13    0.000000E+00    0.968863E+04

                                                      0.212201E-01    Max error
     Iteration number:        11

     =============================================================================================
       Temp     Fermi        DOS        Population SR                 Conductivity 
        [K]      [eV]    [states/Ry] [carriers per cell]               [Ohm.cm]^-1 
     =============================================================================================

      100.000  11.4498   0.37826E+01   0.15987E-13    0.110039E+06    0.512867E-13    0.615440E-12
                                                      0.512867E-13    0.110039E+06    0.512867E-13
                                                      0.000000E+00    0.512867E-13    0.110039E+06
      200.000  11.4799   0.26985E+01  -0.55513E-15    0.246179E+05   -0.256433E-13   -0.128217E-12
                                                      0.128217E-13    0.246179E+05   -0.128217E-13
                                                      0.769300E-13    0.000000E+00    0.246179E+05
      300.000  11.5133   0.16747E+01  -0.33339E-15    0.968863E+04    0.641083E-14   -0.115395E-12
                                                      0.128217E-13    0.968863E+04   -0.128217E-13
                                                     -0.384400E-13   -0.641083E-14    0.968863E+04

                                                      0.537484E-02    Max error
     Iteration number:        12

     =============================================================================================
       Temp     Fermi        DOS        Population SR                 Conductivity 
        [K]      [eV]    [states/Ry] [carriers per cell]               [Ohm.cm]^-1 
     =============================================================================================

      100.000  11.4498   0.37826E+01  -0.84703E-21    0.110039E+06   -0.256433E-13   -0.410293E-12
                                                     -0.153860E-12    0.110039E+06    0.512867E-13
                                                     -0.256433E-12   -0.179503E-12    0.110039E+06
      200.000  11.4799   0.26985E+01   0.11095E-15    0.246179E+05    0.384650E-13   -0.769269E-13
                                                     -0.128217E-13    0.246179E+05   -0.128217E-13
                                                     -0.102570E-12    0.128217E-13    0.246179E+05
      300.000  11.5133   0.16747E+01  -0.94401E-15    0.968863E+04    0.641083E-14    0.320542E-13
                                                     -0.641083E-14    0.968863E+04   -0.641083E-14
                                                      0.256183E-13   -0.641083E-14    0.968863E+04

                                                      0.152240E-02    Max error
     Iteration number:        13

     =============================================================================================
       Temp     Fermi        DOS        Population SR                 Conductivity 
        [K]      [eV]    [states/Ry] [carriers per cell]               [Ohm.cm]^-1 
     =============================================================================================

      100.000  11.4498   0.37826E+01  -0.24425E-14    0.110039E+06   -0.179503E-12    0.148731E-11
                                                      0.000000E+00    0.110039E+06    0.000000E+00
                                                      0.133345E-11   -0.256433E-13    0.110039E+06
      200.000  11.4799   0.26985E+01  -0.99920E-15    0.246179E+05   -0.512867E-13   -0.769269E-13
                                                     -0.256433E-13    0.246179E+05    0.000000E+00
                                                     -0.641052E-13    0.000000E+00    0.246179E+05
      300.000  11.5133   0.16747E+01  -0.43368E-18    0.968863E+04    0.128217E-13    0.897517E-13
                                                     -0.641083E-14    0.968863E+04   -0.641083E-14
                                                      0.192075E-13    0.000000E+00    0.968863E+04

                                                      0.403308E-03    Max error
     Iteration number:        14

     =============================================================================================
       Temp     Fermi        DOS        Population SR                 Conductivity 
        [K]      [eV]    [states/Ry] [carriers per cell]               [Ohm.cm]^-1 
     =============================================================================================

      100.000  11.4498   0.37826E+01  -0.88818E-15    0.110039E+06   -0.769300E-13    0.718013E-12
                                                      0.000000E+00    0.110039E+06    0.000000E+00
                                                      0.923160E-12    0.769300E-13    0.110039E+06
      200.000  11.4799   0.26985E+01  -0.22206E-15    0.246179E+05   -0.128217E-13    0.384619E-13
                                                      0.000000E+00    0.246179E+05    0.000000E+00
                                                     -0.384681E-13    0.128217E-13    0.246179E+05
      300.000  11.5133   0.16747E+01   0.10547E-14    0.968863E+04    0.000000E+00    0.832908E-13
                                                      0.000000E+00    0.968863E+04    0.128217E-13
                                                      0.704691E-13    0.000000E+00    0.968863E+04

                                                      0.111167E-03    Max error
     Iteration number:        15

     =============================================================================================
       Temp     Fermi        DOS        Population SR                 Conductivity 
        [K]      [eV]    [states/Ry] [carriers per cell]               [Ohm.cm]^-1 
     =============================================================================================

      100.000  11.4498   0.37826E+01  -0.13545E-13    0.110039E+06    0.000000E+00    0.666727E-12
                                                      0.102573E-12    0.110039E+06    0.000000E+00
                                                      0.128217E-11    0.000000E+00    0.110039E+06
      200.000  11.4799   0.26985E+01  -0.14433E-14    0.246179E+05   -0.256433E-13   -0.320545E-12
                                                      0.384650E-13    0.246179E+05    0.128217E-13
                                                     -0.269258E-12   -0.256433E-13    0.246179E+05
      300.000  11.5133   0.16747E+01   0.55403E-16    0.968863E+04   -0.128217E-13   -0.129218E-13
                                                      0.192325E-13    0.968863E+04   -0.641083E-14
                                                      0.384900E-13    0.000000E+00    0.968863E+04

                                                      0.299252E-04    Max error
     Iteration number:        16

     =============================================================================================
       Temp     Fermi        DOS        Population SR                 Conductivity 
        [K]      [eV]    [states/Ry] [carriers per cell]               [Ohm.cm]^-1 
     =============================================================================================

      100.000  11.4498   0.37826E+01  -0.19984E-14    0.110039E+06   -0.512867E-13    0.102573E-12
                                                      0.000000E+00    0.110039E+06    0.000000E+00
                                                     -0.512869E-13    0.512867E-13    0.110039E+06
      200.000  11.4799   0.26985E+01  -0.12212E-14    0.246179E+05   -0.128217E-13   -0.115392E-12
                                                     -0.128217E-13    0.246179E+05   -0.128217E-13
                                                     -0.512835E-13    0.128217E-13    0.246179E+05
      300.000  11.5133   0.16747E+01   0.11102E-15    0.968863E+04   -0.641083E-14   -0.577476E-13
                                                      0.000000E+00    0.968863E+04    0.128217E-13
                                                     -0.321043E-13    0.641083E-14    0.968863E+04

                                                      0.816971E-05    Max error
     Iteration number:        17

     =============================================================================================
       Temp     Fermi        DOS        Population SR                 Conductivity 
        [K]      [eV]    [states/Ry] [carriers per cell]               [Ohm.cm]^-1 
     =============================================================================================

      100.000  11.4498   0.37826E+01   0.22205E-15    0.110039E+06    0.512867E-13   -0.256433E-12
                                                      0.000000E+00    0.110039E+06    0.000000E+00
                                                     -0.307720E-12    0.512867E-13    0.110039E+06
      200.000  11.4799   0.26985E+01   0.22204E-15    0.246179E+05   -0.512867E-13   -0.192328E-12
                                                     -0.128217E-13    0.246179E+05    0.128217E-13
                                                     -0.769331E-13    0.256433E-13    0.246179E+05
      300.000  11.5133   0.16747E+01   0.66581E-15    0.968863E+04    0.641083E-14   -0.576474E-13
                                                      0.128217E-13    0.968863E+04    0.128217E-13
                                                     -0.576224E-13   -0.641083E-14    0.968863E+04

                                                      0.221180E-05    Max error
     Iteration number:        18

     =============================================================================================
       Temp     Fermi        DOS        Population SR                 Conductivity 
        [K]      [eV]    [states/Ry] [carriers per cell]               [Ohm.cm]^-1 
     =============================================================================================

      100.000  11.4498   0.37826E+01   0.62173E-14    0.110039E+06   -0.205147E-12    0.718013E-12
                                                      0.102573E-12    0.110039E+06    0.102573E-12
                                                      0.410293E-12    0.000000E+00    0.110039E+06
      200.000  11.4799   0.26985E+01  -0.55510E-15    0.246179E+05    0.000000E+00   -0.769300E-13
                                                      0.256433E-13    0.246179E+05    0.256433E-13
                                                      0.256433E-13    0.000000E+00    0.246179E+05
      300.000  11.5133   0.16747E+01   0.44355E-15    0.968863E+04    0.641083E-14    0.641083E-14
                                                      0.000000E+00    0.968863E+04    0.000000E+00
                                                      0.576725E-13    0.641083E-14    0.968863E+04

                                                      0.601693E-06    Max error

     Unfolding on the coarse grid
     elphon_wrap  :      0.01s CPU      0.01s WALL (       1 calls)

     INITIALIZATION: 




     Electron-Phonon interpolation
     ephwann      :      4.20s CPU      5.83s WALL (       1 calls)
     ep-interp    :      3.43s CPU      5.06s WALL (     723 calls)

     DynW2B       :      0.02s CPU      0.03s WALL (     723 calls)
     HamW2B       :      1.03s CPU      1.12s WALL (   68103 calls)
     ephW2Bp      :      0.77s CPU      0.91s WALL (     723 calls)
     ephW2B       :      0.02s CPU      0.02s WALL (    1224 calls)
     print_ibte   :      0.20s CPU      1.53s WALL (     723 calls)
     vmewan2bloch :      0.07s CPU      0.08s WALL (    2448 calls)
     vmewan2bloch :      0.07s CPU      0.08s WALL (    2448 calls)


     Total program execution
     EPW          :      4.21s CPU      5.85s WALL

     ===============================================================================
     The functionality-dependent EPW.bib file was created with suggested citations. 
     Please consider citing the papers listed in EPW.bib.                           
     ===============================================================================
                                                                                    
